src/com/dropbox/core/DbxAccountInfo.java

```diff
@@ -18,14 +18,22 @@
     public final String country;
     public final String referralLink;
     public final Quota quota;
+    public final String email;
+    public final NameDetails nameDetails;
+    public final Boolean emailVerified;
```

Comment:
@cakoose - Thanks for your feedback! Have updated the pull request with the change.